<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileDialog</class>
 <widget class="QDialog" name="FileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>608</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Новый файл</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Содержание</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="filenameTxt">
         <property name="placeholderText">
          <string>Имя файла</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPlainTextEdit" name="fileContentTxt">
         <property name="placeholderText">
          <string>Содержание файла</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Свойства</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QTableWidget" name="propertiesTable">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="textElideMode">
          <enum>Qt::ElideRight</enum>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>ID владельца</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>ID владеющей группы</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Размер (в байтах)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Дата создания</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Дата модификации</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string/>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="rightsTable">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::AdjustToContents</enum>
         </property>
         <row>
          <property name="text">
           <string>Владелец</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Владеющая группа</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Все остальные</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string>Чтение</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Запись</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Исполнение</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="0" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="1" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="1">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
         <item row="2" column="2">
          <property name="text">
           <string/>
          </property>
          <property name="checkState">
           <enum>Unchecked</enum>
          </property>
          <property name="flags">
           <set>ItemIsUserCheckable|ItemIsEnabled</set>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QCheckBox" name="suidChb">
           <property name="text">
            <string>SUID</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="sgidChb">
           <property name="text">
            <string>SGID</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Сохранить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
